src/lib/pages/home-page-data.ts

```diff
@@ -0,0 +1,41 @@
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
+
+export async function loadHomePageData() {
+  const slugs = {
+    jumbotron: 'primary-jumbotron-cloudflare-workers-intro',
+  }
+  const data = await sanityClient.fetch(
+    groq`
+      {
+        'jumbotron': ${jumbotronQuery},
+      }
+  `,
+    slugs,
+  )
+
+  return data
+}
+
+const jumbotronQuery = groq`
+*[slug.current == $jumbotron][0]{
+  name,
+  title,
+  description,
+  summary,
+  byline, 
+  meta,
+  path,
+  'slug': resources[][0]->_id,
+  'instructor': collaborators[]->[role == 'instructor'][0]{
+    title,
+    'slug': person->slug.current,
```

Comment:
projection is nice, I wonder if they have something like fragments for this type of thing